inclusive-design/wecount.inclusivedesign.ca
# Add EDS jurisdiction review resources to WeCount Resources (resolves #1114)

## Description



See #1114 for details.

## Steps to test

1. Go to resources page
2. Check there's new resource category for ICT standards
3. Check if filtering by the new categories shows results with ICT tags

## Related issues

Resolves #1114
